Resources/FAQ/Jobs-Discord-Asked-Questions.md

```diff
@@ -89,18 +119,20 @@ This is a loaded question, because it can mean a lot of things. Another plugin c
 
 ---
 
-### What are the placeholders for Jobs plugin?
+### Where can I find a list of all Jobs Reborn placeholders?
+
+Type: `jobs placeholders` to get the complete list in the console.
 
-In the console, type: `jobs placeholders` to get the full list.
+You can also do this in-game with the command `/jobs placeholders`
```

Comment:
This repeat information in an unneeded way: maybe remove it, and alter the original to just say to run the command anywhere with permission